code/datums/ruins/lavaland.dm

```diff
@@ -41,7 +41,7 @@
 	description = "The creators of these vaults were a highly advanced and benevolent race, and launched many into the stars, hoping to aid fledgling civilizations. \
 	However, all the inhabitants seem to do is grow drugs and guns."
 	suffix = "lavaland_surface_seed_vault.dmm"
-	cost = 10
+	cost = 5
```

Comment:
_Doubled_ is a lot. Maybe put it to like 7 / 8

Reply:
Done and done!